.gitignore

```diff
@@ -6,3 +6,8 @@ tests/backstop/
 
 # Built css and js assets
 assets/build/*
+
+# Playwright
+/e2e-results/
+/e2e-report/
+/playwright/.cache/
```

Comment:
do we still need the playwright directory here ?

Reply:
Good question, I'm not really sure what it's for, it's added there by default during setup but I can't find anything about it in the docs 🤷‍♀️